plot/steer_aggregate_and_plot_observables.py

```diff
@@ -216,26 +448,121 @@ def main():
+                            f'analysis',
+                            f'--run_number={run_number}', 
+                            f'--run_info_dir={run_info_download_location}', 
+                            f'--final_state_hadrons_dir={final_state_hadrons_download_location}', 
+                            f'--output_dir={reananalysis_output_location}', 
+                            f'--analysis_facility_name={local_analysis_facility}', 
+                            f'--download_facility_name={facility}',
+                            f'--container_path={analyis_container_path}', 
+                            f'--n_tasks={ntasks}', 
+                            f'--max_n_slurm_jobs={max_n_slurm_jobs}', 
+                            f'--jetscape_analysis_dir={jetscape_analysis_dir}']
+                subprocess.run(arg_list, check=True, shell=False)
+                if delete_final_state_hadrons_after_analysis:
+                    # safely delete final_state_hadron_dir, make sure the path contains final_state_hadrons_per_run so we don't by accident delete "/" or "/home" .... 
+
+                    if os.path.exists(final_state_hadron_dir) and "final_state_hadrons_per_run" in final_state_hadron_dir:
+                        # remove all files in the directory that contain final_state_hadrons and parquet in name
+                        # note that we don't ask for endging parquet since sometimes incomplete files have weird ending
+                        print(f'Deleting all parquet files in {final_state_hadron_dir} as requested...')
+                        for f in os.listdir(final_state_hadron_dir):
+                            if "final_state_hadrons" in f and "parquet" in f:
+                                os.remove(os.path.join(final_state_hadron_dir, f))
+                            # remove DONE file
+                            if "DONE" in f:
+                                os.remove(os.path.join(final_state_hadron_dir, f))
+                        # now we can delete the directory
+                        os.rmdir(final_state_hadron_dir)
+
+                # TODO upload analysis results / histos to OSN in the future
+                
+
+
+        print('Done!')
+    
 
     #-----------------------------------------------------------------
     # Merge for each run into a single histogram per run
     #-----------------------------------------------------------------
     if merge_histograms:
 
         for facility in facilities:
+            run_counter = 0
             for run in runs[facility]:
+                run_counter += 1
+
+                # TODO make counter "Processing run X out of Y at facility Z"
+                print(f'Processing run {run_counter} out of {len(runs[facility])} at facility {facility}...')
 
-                outputdir = os.path.join(local_base_outputdir, f'histograms_per_run/{facility}/{run}')
+                outputdir = os.path.join(reananalysis_output_location_base, f'{facility}/{run}')
```

Comment:
Will this work if we're doing the standard aggregation? Or was this variable renamed?

Reply:
It is the same path as before
```
reananalysis_output_location_base = os.path.join(local_base_outputdir, 'histograms_per_run')
```
However I agree with you in this contect the name is confusing so I reverted to the old way of writing it